lib/facebook.ex

```diff
@@ -187,37 +187,52 @@ defmodule Facebook do
   end
 
   @doc """
-  Feed of posts for the provided page_id.
-  The maximum posts returned is 25, which is the facebook's default.
+  Gets the feed of posts (including status updates) and links published by this
+  page, or by others on this page.
 
-  ## Example
-      iex> Facebook.pageFeed("CocaColaMx", "<Your Token>")
+  This function can retrieve the four types:
+    * feed
+    * posts
+    * promotable posts (*Admin permission needed*)
+    * tagged posts
+
+  A scope must be provided. It is a string, which represents the type of feed.
+
+  *A limit of posts may be given. The maximum number that must be provided is
+  100.*
+
+  ## Examples
+      iex> Facebook.pageFeed(:posts, "CocaColaMx", "<Your Token>")
+      iex> Facebook.pageFeed(:tagged, "CocaColaMx", "<Your Token>", 55)
+      iex> Facebook.pageFeed(:promotable, "CocaColaMx", "<Your Token>")
+      iex> Facebook.pageFeed(:feed, "CocaColaMx", "<Your Token>", 55, "id,name")
 
   See: https://developers.facebook.com/docs/graph-api/reference/page/feed
   """
-  def pageFeed(page_id, access_token) do
-    pageFeed(page_id, access_token, 25)
+  def pageFeed(scope, page_id, access_token, limit \\ 25, fields \\ "") do
+    case scope do
+      :feed -> feed("feed", page_id, access_token, limit, fields)
+      :posts -> feed("posts", page_id, access_token, limit, fields)
+      :promotable -> feed("promotable_posts", page_id, access_token, limit, fields)
+      :tagged -> feed("tagged", page_id, access_token, limit, fields)
+      _ -> %{"error" => %{"message" => "Unknown type of feed"}}
```

Comment:
Could be done easier by just checking if the scope is one of the 4 allowed entries (or not checking it at all, facebook will tell you it's wrong anyway) and passing the scope just like it is forward. 
Atoms will automatically get converted to a string when json encoding it, so that should be no problem.